readme.md

```diff
@@ -1,4 +1,4 @@
-# wrap-ansi [![Build Status](https://travis-ci.org/chalk/wrap-ansi.svg?branch=master)](https://travis-ci.org/chalk/wrap-ansi)
+# wrap-ansi [![Build Status](https://travis-ci.org/chalk/wrap-ansi.svg?branch=master)](https://travis-ci.org/chalk/wrap-ansi) [![Coverage Status](https://coveralls.io/repos/chalk/wrap-ansi/badge.svg?branch=master)](https://coveralls.io/r/chalk/wrap-ansi?branch=master)
```

Comment:
This branch is forked off of #4, so once #4 is landed these changes will not be part of this pull -- by forking from the code-coverage branch, I was able to see coverage as I worked on tests for this branch.